### PR TITLE
Add Wondershare PDFelement 8.2.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19778)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@msftbot merge this commit

File: manifests/w/WondershareSoftwareCoLtd/PDFelement/8.0/WondershareSoftwareCoLtd.PDFelement.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "WondershareSoftwareCoLtd.PDFelement"
+PackageVersion: "8.0"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare PDFelement"
```
Comment (OfficialEsco) [inline manifests/w/WondershareSoftwareCoLtd/PDFelement/8.0/WondershareSoftwareCoLtd.PDFelement.yaml]:
```suggestion
PackageIdentifier: Wondershare.PDFelement
PackageVersion: 8.2.2
PackageLocale: en-US
Publisher: Wondershare
PackageName: Wondershare PDFelement
```
image

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/w/Wondershare/PDFelement/Wondershare.PDFelement.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "Wondershare.PDFelement"
+PackageVersion: "8.0"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare PDFelement"
```
Comment (OfficialEsco) [inline manifests/w/Wondershare/PDFelement/Wondershare.PDFelement.yaml:5]:
```suggestion
PackageIdentifier: Wondershare.PDFelement
PackageVersion: 8.2.2
PackageLocale: en-US
Publisher: Wondershare
PackageName: Wondershare PDFelement
```
image

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco please approve my changes

File: manifests/w/Wondershare/PDFelement/8.2.2/Wondershare.PDFelement.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: Wondershare.PDFelement
+PackageVersion: 8.2.2
+PackageLocale: en-US
+Publisher: Wondershare
+PackageName: Wondershare PDFelement
+License: "EULA"
```
Comment (quhxl) [inline manifests/w/Wondershare/PDFelement/8.2.2/Wondershare.PDFelement.yaml:6]:
```suggestion
License: Copyright © 2021 Wondershare.
LicenseUrl: https://www.wondershare.com/company/end-user-license-agreement.html
```

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/w/Wondershare/PDFelement/Wondershare.PDFelement.yaml (hunk 2 above)
Comment (vedantmgoyal2009) [reply manifests/w/Wondershare/PDFelement/Wondershare.PDFelement.yaml]: from where this has been taken?

**Comment by quhxl** [comment]
Why do you always use quotation marks? You don't need them, only when you use these characters: `:, {, }, [, ], ,, &, *, #, ?, |, -, <, >, =, !, %, @, \`

**Comment by vedantmgoyal2009** [comment]
> Why do you always use quotation marks? You don't need them, only when you use these characters: `:, {, }, [, ], ,, &, *, #, ?, |, -, <, >, =, !, %, @, \`

sorry

**Review by OfficialEsco — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
Thanks!

**Pull request merged**